installer/templates/phx_single/config/config.exs

```diff
@@ -17,7 +17,10 @@ config :<%= app_name %>, <%= endpoint_module %>,
   url: [host: "localhost"],
   secret_key_base: "<%= secret_key_base %>",
   render_errors: [view: <%= web_namespace %>.ErrorView, accepts: ~w(<%= if html do %>html <% end %>json), layout: false],
-  pubsub: [name: <%= app_module %>.PubSub, adapter: Phoenix.PubSub.PG2]
+  pubsub: [name: <%= app_module %>.PubSub, adapter: Phoenix.PubSub.PG2]<%= if live do %>,
+  live_view: [
+    signing_salt: "<%= live_view_signing_salt %>"
+  ]<% end %>
```

Comment:
See comment here
https://github.com/phoenixframework/phoenix/pull/3376/files#r275770071

The live_view_signing_salt might completely go away.